### PR TITLE
Remove "glamorous-native"

### PR DESCRIPTION
**Pull request by hawkrives**
I wanted to remove the `glamorous-native` dependency. It's unmaintained, and one of the sources of warnings that we see in the console. 

I'll deal with the react-markdown stuff in a separate PR.

### CODE REVIEW

File: modules/markdown/markdown.tsx [hunk 1]
```diff
@@ -22,13 +23,20 @@ import {Link} from './link'
 import {Image} from './image'
 import {List, ListItem} from './list'
 
+const styles = StyleSheet.create({
+	// eslint-disable-next-line react-native/no-color-literals
```
Comment (hawkrives) [inline modules/markdown/markdown.tsx]: I don't know why line 27 is the right spot for this comment, when it affects line 31 or something, but I've learned not to question ESLint.

File: modules/markdown/markdown.tsx [hunk 2]
```diff
@@ -37,7 +45,7 @@ export type MarkdownProps = {
 		CodeBlock?: StyleProp<TextStyle>
 		Emph?: StyleProp<TextStyle>
 		Heading?: StyleProp<TextStyle>
-		Image?: StyleProp<ViewStyle>
+		Image?: StyleProp<ImageStyle>
```
Comment (hawkrives) [inline modules/markdown/markdown.tsx:48]: While this is "more correct" now… let's not try actually rendering anything, I'm pretty sure that's broken. (The props of img to Image are pretty different)

File: modules/markdown/markdown.tsx (hunk 1 above)
Comment (drewvolz) [reply modules/markdown/markdown.tsx]: Do we want to instead import c.black from frogpond/colors?

File: source/views/settings/screens/credits.tsx [hunk 3]
```diff
@@ -27,45 +56,27 @@ const markdownStyles: MarkdownProps['styles'] = {
...
-		android: material.body1Object,
-	}),
-	textAlign: 'center',
-})
+const Contributors = (props: TextProps) => (
+	<Text {...props} style={[styles.contributors, props.style]} />
+)
 
 const formatPeopleList = (arr: Array<string>) =>
 	arr.map((w) => w.replace(' ', ' ')).join(' • ')
```
Comment (drewvolz) [inline source/views/settings/screens/credits.tsx:76]: What is going on with the replace call?

**Review by drewvolz — APPROVED** [review]

File: source/views/settings/screens/credits.tsx (hunk 3 above)
Comment (hawkrives) [reply source/views/settings/screens/credits.tsx:76]: I believe it replaces spaces with non-breaking spaces, so that people's names don't wrap?

**Review by rye — APPROVED** [review]

**Pull request merged**